lib/lotus/generators/model/.gitkeep.tt

```diff
@@ -0,0 +1 @@
+#
```

Comment:
@jodosha it was due to the fact the generator cli https://github.com/nguyenngoc2505/lotus/blob/issues/198/lib/lotus/commands/generate.rb#L35, I am suggesting to her that if `type == "model"`, just skip setting that `@source`